pkg/syncer/util.go

```diff
@@ -185,37 +185,37 @@ func IsValidVolume(ctx context.Context, volume v1.Volume, pod *v1.Pod,
 func fullSyncGetQueryResults(ctx context.Context, volumeIds []cnstypes.CnsVolumeId, clusterID string,
 	volumeManager volumes.Manager, metadataSyncer *metadataSyncInformer) ([]*cnstypes.CnsQueryResult, error) {
 	log := logger.GetLogger(ctx)
-	log.Debugf("FullSync: fullSyncGetQueryResults is called with volumeIds %v for clusterID %s", volumeIds, clusterID)
-	queryFilter := cnstypes.CnsQueryFilter{
-		VolumeIds: volumeIds,
-		Cursor: &cnstypes.CnsCursor{
-			Offset: 0,
-			Limit:  queryVolumeLimit,
-		},
-	}
-	if clusterID != "" {
-		queryFilter.ContainerClusterIds = []string{clusterID}
+	log.Debugf("FullSync: fullSyncGetQueryResults is called with volumeIds %v for clusterID %s",
+		volumeIds, clusterID)
+	useQueryVolumeAsync := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.AsyncQueryVolume)
+	var volumeIdsBatchesToFilter [][]cnstypes.CnsVolumeId
+	for i := 0; i < len(volumeIds); i += 1000 {
```

Comment:
Replace 1000 with some variable, may be use "common.queryVolumeLimit"

Reply:
done.